docs/metrics.md

```diff
@@ -15,3 +15,11 @@ When [Caddy metrics](https://caddyserver.com/docs/metrics) are enabled, FrankenP
 - `frankenphp_worker_queue_depth{worker="[worker_name]"}`: The number of queued requests.
 
 For worker metrics, the `[worker_name]` placeholder is replaced by the worker name in the Caddyfile, otherwise absolute path of worker file will be used.
+
+Here is how to enable the metrics using environment variables:
+
+```sh
+CADDY_GLOBAL_OPTIONS="servers {
+  metrics
+}
```

Comment:
```suggestion
CADDY_GLOBAL_OPTIONS="
  metrics
"
```

https://caddyserver.com/docs/metrics#enabling-metrics

Reply:
@IndraGunawan have you tried it this way?

Reply:
I used metrics in my Caddyfile global options yesterday. What version on FrankenPHP are you using?

Reply:
1.3.3 Caddy v2.8.4

Reply:
```
/app # cat /etc/caddy/Caddyfile 
{
        {$CADDY_GLOBAL_OPTIONS}

        frankenphp {
                #worker /path/to/your/worker.php
                {$FRANKENPHP_CONFIG}
        }

        # https://caddyserver.com/docs/caddyfile/directives#sorting-algorithm
        order mercure after encode
        order vulcain after reverse_proxy
        order php_server before file_server
        order php before file_server
}
```

And in my docker-compose environment I have this:

```
    environment:
      CADDY_GLOBAL_OPTIONS: |
        servers {
          metrics
          timeouts {
            read_body 5m
            read_header 5s
            write 5m
            idle 5m
          }
        }
```

Reply:
Oh. Damn. I've verified with dated version. Sorry.

Reply:
> ```
> /app # cat /etc/caddy/Caddyfile 
> {
>         {$CADDY_GLOBAL_OPTIONS}
> 
>         frankenphp {
>                 #worker /path/to/your/worker.php
>                 {$FRANKENPHP_CONFIG}
>         }
> 
>         # https://caddyserver.com/docs/caddyfile/directives#sorting-algorithm
>         order mercure after encode
>         order vulcain after reverse_proxy
>         order php_server before file_server
>         order php before file_server
> }
> ```

FYI, you can already get rid of all the order directives. They're all built in by default now.